client/tests/integration_test.rs

```diff
@@ -606,15 +605,19 @@ mod get_item {
     #[tokio::test(flavor = "multi_thread")]
     async fn should_fail_with_empty_key() {
         let server_handle = MockServerHandle::spawn::<GetItemParams>(GetItem::METHOD);
-        assert!(matches!(
-            server_handle
-                .get_item("<invalid state root hash>", "", "")
-                .await,
-            Err(Error::CryptoError {
-                context: "state_root_hash",
-                error: CryptoError::FromHex(FromHexError::OddLength)
-            })
-        ));
+        let blah = server_handle
+            .get_item("<invalid state root hash>", "", "")
+            .await;
+        println!("{:?}", blah);
+        // assert!(matches!(
+        //     server_handle
+        //         .get_item("<invalid state root hash>", "", "")
+        //         .await,
+        //     Err(Error::CryptoError {
+        //         context: "state_root_hash",
+        //         error: CryptoError::FromHex(base16::DecodeError::InvalidLength { length: 25 })
+        //     })
+        // ));
```

Comment:
@nilz3ro What happened here?

Reply:
Oh, I was having a look around and I forgot to remove this.